.codecov.yml

```diff
@@ -1,5 +1,8 @@
 codecov:
-  require_ci_to_pass: yes
+  require_ci_to_pass: no
```

Comment:
???

Reply:
It means we get coverage data after py27 and py36 rather than waiting for windows and mac to finish.
And we get coverage info even if something fails, which is also useful to know.